src/Illuminate/View/ViewException.php

```diff
@@ -3,20 +3,22 @@
 namespace Illuminate\View;
 
 use ErrorException;
+use Illuminate\Container\Container;
+use Illuminate\Support\Reflector;
 
 class ViewException extends ErrorException
 {
     /**
      * Report the exception.
      *
-     * @return void
+     * @return bool|null
      */
     public function report()
     {
         $exception = $this->getPrevious();
 
-        if ($exception && method_exists($exception, 'report')) {
-            $exception->report();
+        if (Reflector::isCallable($reportCallable = [$exception, 'report'])) {
+            return Container::getInstance()->call($reportCallable);
         }
```

Comment:
I think this also needs to return `false` if the previous exception doesn't have a `report` method. Otherwise exceptions that don't implement `report` will still not be logged:

```suggestion
        if (Reflector::isCallable($reportCallable = [$exception, 'report'])) {
            return Container::getInstance()->call($reportCallable);
        }

        return false
```

Reply:
Good catch 👍